apps/wallets/smart-wallet/expo/app/index.tsx

```diff
@@ -15,7 +16,7 @@ export default function Index() {
     const { experimental_customAuth } = useCrossmint();
     const loggedInUserEmail = experimental_customAuth?.email ?? null;
     const { wallet, getOrCreateWallet, status: walletStatus } = useWallet();
-    const { needsAuth, sendEmailWithOtp, verifyOtp } = useWalletEmailSigner();
+    const { needsAuth, sendOtp, verifyOtp } = useWalletOtpSigner();
```

Comment:
needsAuth -- dont use Auth, name it something like isInitialized (and invert it)